pkg/importer/dataStream.go

```diff
@@ -307,6 +312,29 @@ func SaveStream(stream io.ReadCloser, dest string) (int64, error) {
 	return ds.Size, nil
 }
 
+// ResizeImage resizes the images to match the requested size. Sometimes provisioners misbehave and the available space
+// is not the same as the requested space. For those situations we compare the available space to the requested space and
+// use the smallest of the two values.
+func ResizeImage(dest, imageSize string) error {
```

Comment:
Is this being called somewhere?

Reply:
Whoops, you are right it is not, but obviously it should, making a patch to fix now.